samples/apps/copilot-chat-app/webapp/src/components/chat/chat-list/ChatListItem.tsx

```diff
@@ -86,6 +87,11 @@ const useClasses = makeStyles({
     selected: {
         backgroundColor: tokens.colorNeutralBackground1,
     },
+    protectedIcon: {
+        color: '#6BB700',
```

Comment:
I'm wondering if we need to start adjusting colors depending on light/dark mode?